test/framework/easyblock.py

```diff
@@ -611,7 +611,7 @@ def test_exclude_path_to_top_of_module_tree(self):
             for imkl_dep in excluded_deps:
                 tup = (imkl_dep, modfile_path, modtxt)
                 failmsg = "No 'module load' statement found for '%s' not found in module %s: %s" % tup
-                self.assertFalse(re.search("module load %s" % imkl_dep, modtxt), failmsg)
+                self.assertFalse(re.search('module load "%s"' % imkl_dep, modtxt), failmsg)
```

Comment:
Now that I've looked at this again, I fully agree.  I have no clue what led me to this change...

But... This test fails since https://github.com/geimer/easybuild-framework/commit/2335cce791cad9fb251c9ffaf4be94fa9ab74ff9 and I have no idea yet why.